.github/.cSpellWords.txt

```diff
@@ -0,0 +1,25 @@
+CBMC
+CBOR
+CMOCK
+CMock
+Cmock
+Coverity
+DCMOCK
+DDisable
+DNDEBUG
+DUNITY
+MISRA
+MQTT
+Misra
+Wunused
+abcdefghijklmno
```

Comment:
What is the purpose of this and acceptedklmno?

Reply:
It is used in the test buffer `TEST_CLASSIC_TOPIC_BUFFER_INITIALIZE` in [test/unit-test/shadow_utest.c:257](https://github.com/aws/Device-Shadow-for-AWS-IoT-embedded-sdk/blob/main/test/unit-test/shadow_utest.c#L257)